lib/vehicle.rb

```diff
@@ -22,4 +26,19 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def set_date
+    @registration_date = Date.today
+  end
+
+  def give_plate
```

Comment:
I'd call this method something like `set_plate_type`

Reply:
Also, I do not see tests for the set_date and give_plate methods in the vehicle_spec. Remember to always add tests for methods we add to classes.

Reply:
Yes. I missed that.